entropy/windows/Raaz/Entropy.hsc

```diff
@@ -30,18 +31,19 @@ type HCRYPTPROV = Ptr ()
 foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptGenRandom"
     c_CryptGenRandom :: HCRYPTPROV -> Word32 -> Ptr Word8 -> IO Bool
 
-foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptAcquireContext"
+foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptAcquireContextW"
     c_CryptAcquireContext :: Ptr HCRYPTPROV -> CWString -> CWString
                           -> Word32 -> Word32 -> IO Bool
 
 foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptReleaseContext"
     c_CryptReleaseContext :: HCRYPTPROV -> Word32 -> IO Bool
 
 -- | Cache the crypto context so we don't have to create it on each call.
-cryptoContext :: ForeignPtr (HCRYPTPROV)
+cryptoContext :: ForeignPtr HCRYPTPROV
```

Comment:
This library exposes the command raaz which can be used to generate random bytes. So you may try the windows equivalent of this 

```bash
stack exec raaz rand -- 10 | od -x
```
and inspect the output. Surprisingly this simple test caught an error on the OpenBSD system. There is also the script in the subdirectory  `./spec/prgquality.sh` which runs the dieharder on the output of stack exec raaz rand. If dieharder is available on windows you can try that (If you do not mind that is).